pkg/kubelet/logs/container_log_manager.go

```diff
@@ -412,7 +412,8 @@ func (c *containerLogManager) rotateLatestLog(id, log string) error {
 			// This shouldn't happen.
 			// Report an error if this happens, because we will lose original
 			// log.
-			klog.Errorf("Failed to rename rotated log %q back to %q: %v, reopen container log error: %v", rotated, log, renameErr, err)
+			klog.ErrorS(renameErr, "Failed to rename rotated log back to original, after reopen container log error",
+				"path", rotated, "originalPath", log, "reopenErr", err)
```

Comment:
/cc @serathius 
on how we want to do multiple errors

Reply:
Ah, we had this one already: https://github.com/kubernetes/kubernetes/pull/99848#discussion_r588870925

/close

@krzysiekg thank you for this PR! it conflicts with an earlier one for the same file. would you be able to pick up something else from https://github.com/kubernetes/kubernetes/issues/98976#issuecomment-790927982 ?

Reply:
No problem! My bad, I missed this on the list. I'm already working on another package.